### DISCUSSION TITLE
1.4.0.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#273] [Обновления]
## Список изменений
- Исправлена работа перевода, если длина видео не была получена
- Исправлен двойной префикс в некоторых ошибках консоль логов
- Исправлена работа настроек в YouTube Embed после поломки в 1.4.0
- *Добавлен protobuf для пинга стримов
- *Обновлен protobuf для запроса перевода стримов
- *Protobuf для проверки стрима на вайтлист был помечен как более неиспользуемый Яндексом (не зависимо от его ответа запрос на перевод стрима будет проходить)

`*` - означает, что это изменение "на будущее"

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.4.0.1...1.4.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.0.2'>1.4.0.2</a>.</em>

**Comment by NullVerdict** [comment]
пмргте
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/f03d728f-868c-4f52-beed-72d4ddc79808)

**Reply by ilyhalight** [reply]
Исправил, надеюсь еще что-то не отвалилось
